airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/form_submissions.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": ["null", "object"],
+    "properties": {
+      "submittedAt": {
+        "type": ["null", "string"],
+        "format": "date-time"
+      },
+      "values": {
+        "type": ["null", "array"],
+        "items": {
+          "type": ["null", "object"],
+          "properties":{
+            "name": {
+              "type": ["null", "string"]
+            },
+            "value": {
+              "type": ["null", "string"]
+            },
+            "objectTypeId": {
+              "value": {
+                "type": ["null", "string"]
```

Comment:
Are u sure, that field which consists `Id` in name has `string` type?

Reply:
Yeah HubSpot returns a value which consists only of this string "0-1". Therefore String.